VTEX - Marketplace Protocol - External Marketplace Mapper.json

```diff
@@ -405,335 +592,252 @@
-                                                "email": "jhon@marketplace.com",
-                                                "phone": "(77) 7777-7777",
-                                                "defaultMessage": ""
-                                            },
-                                            "isConnectionActive": false,
-                                            "affiliateId": "1234"
-                                        }
-                                }
+                                    }
+                                },
+                                "example": {
+                                        "id": 584,
+                                        "name": "Mercado Livre Classic",
+                                        "website": "mercadolivre.com.br",
+                                        "logo": "https://g.foolcdn.com/art/companylogos/square/meli.png",
+                                        "description": "MercadoLibre operates online commerce platforms in Latin America. It operates Mercado Libre Marketplace, an automated online commerce platform that enables businesses, merchants, and individuals to list merchandise and conduct sales and purchases online.",
+                                        "helpUrl": "https://help.vtex.com/pt/tracks/configurar-integracao-do-mercado-livre--2YfvI3Jxe0CGIKoWIGQEIq",
+                                        "segment": "OTHERS",
+                                        "availableCountries": [
+                                            "BRA",
+                                            "ARG"
+                                        ],
+                                        "availableStates": [],
+                                        "type": "EXTERNAL",
+                                        "connectorPageUrl": "https://apps.vtex.com/vtexbr-tiktok-tbp/p",
+                                        "connectorVtexAdminPath": "/bridge/#/settings?openConfig=mercadolivre",
+                                        "connectorApiEndpoint": "portal.vtexcommercestable.com/api/meliintegration/notifications",
+                                        "channelId": "1",
+                                        "isGloballyAvailable": false,
+                                        "ownerVtexAccount": "vtex",
+                                        "isActive": true,
+                                        "isCertified": false,
+                                        "contactInfo": {
+                                            "name": "Jhon",
+                                            "email": "jhon@marketplace.com",
+                                            "phone": "(77) 7777-7777",
+                                            "defaultMessage": ""
+                                        },
+                                        "isConnectionActive": false,
+                                        "affiliateId": "1234"
```

Comment:
Alterar exemplo para algo como CNT, pois:

affiliate ID: the affiliate identifier code, which consists of three consonants, whether they are repeated or not. Vowels are not allowed.

https://help.vtex.com/en/tutorial/configuring-affiliates--tutorials_187